### PR TITLE
Booking Checkout Module

### PR DESCRIPTION
**Pull request by johnsonkwok**
Setup/Schema/Seeding db (code for seeding db still in progress)

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  nightly_price INT NOT NULL,
+  past_week_views INT NOT NULL,
+  cleaning_fee INT NOT NULL,
+  service_fee INT NOT NULL,
+  last_updated DATE,
+  review_count INT NOT NULL,
+  avg_rating INT NOT NULL
+);
+
+CREATE TABLE bookings (
```
Comment (wusihong) [inline db/schema.sql]: how are you dealing with blackout dates per toby's comments?

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*  Execute this file from the command line by typing:
+ *    mysql -u root < schema.sql
+ *  to execute the queries in this file.
+ */
+
+CREATE DATABASE IF NOT EXISTS booking_checkout;
```
Comment (wusihong) [inline db/schema.sql:1]: as discussed, "booking_checkout" seems redundant. suggest either "bookings" or "checkout"

Comment (johnsonkwok) [reply db/schema.sql:1]: changed database name to "booking"

File: db/schema.sql (hunk 1 above)
Comment (johnsonkwok) [reply db/schema.sql]: I added a column "blackout" to the bookings table. It takes a boolean value (0 for false and 1 for true). If a row has a blackout value of "true", it means the room is unavailable, but I will still be able to set that row's dates in the bookings table to indicate whether the room is unavailable even though it is not booked.

**Pull request merged**